bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/config/player-config.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:freeboxos:player">
+		<parameter name="macAddress" type="text" pattern="([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})">
```

Comment:
Please add `required="true"`for this parameter.

Reply:
Of course, do it only if this parameter is mandatory.
But as you defined it as the representation property of your thing, I assume it is.